swagger_server/itm/itm_action_handler.py

```diff
@@ -52,16 +52,18 @@ def _reveal_injuries(self, source: Character, target: Character):
             target.unstructured = source.unstructured_postassess
 
     def __successful_treatment(self, treatment: str, injury_name: str, location: str) -> bool:
-        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, an Internal injuries are currently untreatable.
+        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, Traumatic Brain Injury, Open Abdominal Wound and Internal injuries are currently untreatable.
```

Comment:
Remove "Forehead Scrape," and add an Oxford comma after "Open Abdominal Wound."
Change "Forehead Scrape (Abrasion)" below to "Face Laceration".
Apart from the comma, this is all part of "remove Abrasion," but there's no way you would have known that.